.github/workflows/pythonpackage.yml

```diff
@@ -31,17 +31,16 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install dependencies coverage, codecov, flake8, fastimport, mypy
       run: |
-        python -m pip install --upgrade pip
-        pip install -U pip coverage codecov flake8 fastimport
-    - name: Install mypy
-      run: |
-        pip install -U mypy
+        python -m pip install -r requirements.txt --upgrade
       if: "matrix.python-version != 'pypy3'"
-    - name: Style checks
+    - name: Style checks flake8
       run: |
         python -m flake8
+    - name: Style checks pylint
```

Comment:
I'm not opposed to running pylint for checks that make sense, but I think that should happen as a separate change. There are simply too many code changes to be made for that to be useful.

Reply:
Before I actually merge, I will modify all the code to pass four different checks. After completion, I will continue to assist in writing Pytest. The reason for not writing is that the project currently has a designated Test function, so I don’t Intend to write first.